From 67b5d6bfbee52bd59f3e5dac3e0d552ed572b3b2 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Fri, 21 Dec 2018 11:08:08 +0100
Subject: [PATCH xkeyboard-config] evdev/inet: Fix KEY_KEYBOARD mapping

KEY_KEYBOARD has an evdev-keycode value of 374 not 366. It seems that
with the original addition of the mapping the evdev-keycode value was
mistaken for the X scancode which is 8 higher then the evdev-keycode
since the X scancodes start at 8. What seems to have happened is that
the evdev-keycode was used instead of the X scancode and then 8 was
subtracted for the wrong "// #define KEY_KEYBOARD      366" comment.

This commit corrects the comment to say 374 and corrects the X
scancode being mapped to 382.

Cc: Christian Kellner <christian@kellner.me>
Reviewed-by: Christian Kellner <christian@kellner.me>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 keycodes/evdev | 2 +-
 symbols/inet   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/keycodes/evdev b/keycodes/evdev
index 90b8278..dd2b865 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -285,7 +285,7 @@ default xkb_keycodes "evdev" {
 	<I255> = 255;	// #define KEY_RFKILL              247
 
 	<I372> = 372;   // #define KEY_FAVORITES           364
-	<I374> = 374;   // #define KEY_KEYBOARD            366
+	<I382> = 382;   // #define KEY_KEYBOARD            374
 
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
diff --git a/symbols/inet b/symbols/inet
index 8e65c5e..ae1f6ee 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -219,7 +219,7 @@ xkb_symbols "evdev" {
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
     key <I372>  {       [ XF86Favorites          ]       };
-    key <I374>  {       [ XF86Keyboard           ]       };
+    key <I382>  {       [ XF86Keyboard           ]       };
 
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
-- 
2.21.0

